lib/dotcom/utils/date_time.ex

```diff
@@ -29,6 +29,12 @@ defmodule Dotcom.Utils.DateTime do
   @impl Behaviour
   def now(), do: Timex.now(@timezone)
 
+  @doc """
+  Get the date in the set @timezone.
+  """
+  @impl Behaviour
+  def today(), do: Timex.today(@timezone)
+
```

Comment:
📓 I didn't end up using this, but I figured it is convenient enough to keep in there.